config/default.conf

```diff
@@ -2,6 +2,21 @@ server {
     port 8080
     server_name localhost www.localhost
     error_page 404 pages/error/404.html
+
+    client_max_body_size 1M
+    location /test {
+        cgi .php /path
```

Comment:
Esse to curioso

Reply:
esse aqui é o que vai conectar com o que você fez no último pr (o de receber a request em vários pacotes)